packages/psss/src/utils/calculateWeekStatus.js

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
This is unchanged code. It's just been extracted into it's own file.

Reply:
Thanks for clarifying this!